<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoicesListFilters</class>
 <widget class="QDialog" name="InvoicesListFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Payment List Filters</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F5F8FA;
color: #14171A;
font-family:&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLable">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:18px;
font-weight: 400;</string>
     </property>
     <property name="text">
      <string>Filter List</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="studentWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="studentLabel">
        <property name="styleSheet">
         <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
        </property>
        <property name="text">
         <string>From student:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="studentCombo">
        <property name="styleSheet">
         <string notr="true">QComboBox{
	padding: 6px 4px;
	border: 1px solid #aab8c2;
	border-radius: 2px;
}
QComboBox:hover, QComboBox:focus{
	border: 1px solid #14171a;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="groupWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="groupLabel">
        <property name="styleSheet">
         <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
        </property>
        <property name="text">
         <string>From group:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="groupCombo">
        <property name="styleSheet">
         <string notr="true">QComboBox{
	padding: 6px 4px;
	border: 1px solid #aab8c2;
	border-radius: 2px;
}
QComboBox:hover, QComboBox:focus{
	border: 1px solid #14171a;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="paidUntilWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>70</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="paidUntilLabel">
        <property name="styleSheet">
         <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
        </property>
        <property name="text">
         <string>Paid until:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="paidUntilDate">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">QPushButton {
background: #616161;
border-radius: 2px;
color: #fff;
padding: 8px 16px;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InvoicesListFilters</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InvoicesListFilters</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
